contracts/fast/Crowdfund.sol

```diff
@@ -79,10 +81,12 @@ contract Crowdfund {
     IERC20 token;
     /// @notice An arbitrary reference string to keep track of.
     string ref;
+    /// @notice The cap of the crowdfund if specified.
+    uint256 cap;
   }
 
   /// @notice A version identifier for us to track what's deployed.
-  uint16 public constant VERSION = 2;
+  uint16 public constant VERSION = 3;
```

Comment:
New feature... version bump.

Reply:
Frontend is going to run into some pain here. We should chat.

Reply:
Damn, ok. I'll ping you 👍